src/Web/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -43,5 +43,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Context>BrowseObject</Context>
     </PropertyPageSchema>
   </ItemGroup>
-
-</Project>
\ No newline at end of file
+  
+  <!-- Web specific target to ensure that generated content (css, js, etc.) is copied to output -->
+  <Target Name="_IncludeWebProjectGeneratedContent" BeforeTargets="GetCopyToPublishDirectoryItems" Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultContentItems)' == 'true' ">
```

Comment:
While we're talking about extensibility would I be welcome to open an issue in the main SDK repo about how maybe the SDK could be using the `CallTarget` task for `Publish` instead of `DependsOnTargets` so that a simple `<Target Name="MyTarget" BeforeTargets="Publish">` can work as expected? Or maybe to have the `BeforePublish` and `AfterPublish` overridable targets be hit from the CLI and not just VS publish?